### DISCUSSION TITLE
누워 있을 때 Instant Meeting이 안걸립니다.

### DISCUSSION THREAD

**Discussion by DolDolDol** [metamong-friends/metamong-friends.github.io#24] [제보하기]
누워있는 상대한테는 버블이 안걸리네요. 
그런데 버블이 걸려있을때 누을 수는 있어요. 

일관성 이슈가 있을거 같아서 일단 올려봅니다.

**Comment by ACT-TJ** [comment]
자주.. 누우시나요? 저희도 가끔 누워서 체크인을 하는데 색다르긴 하더라구요^^

눕는 건 저희가 개발 차원에서 다양한 캐릭터 동작을 확인하는 차원에서의 베타 기능으로 넣어두었던 건데
충분히 누운상태에서 연관기능 확인이 되지 않았었네요 ㅠㅠ 

ProjectView 팀만을 위해 open 하고 더 이상 다른 팀에게 소개는 되지 않을 것 같아요 ㅎㅎ
눕는 걸 막게 될 지, 누웠을 때 일관성 이슈를 정리할 지 , 내부적으로 정리해서 피드백 드리도록 하겠습니다.

(재밌게 쓰고계시다면 막기보단, 가볍게 기능정리 정도가 어떨까 싶긴 하네요^^)

**Comment by ACT-TJ** [comment]
누워 있는 동작이 정식 Action 으로 들어가게 되면 전수검사를 통해서 캐릭터의 액션들이 정상적으로
동작되도록 하겠습니다.^^ 해당이슈는 Close 처리하겠습니다. 관심 너무 감사합니다.!
